.ci/Jenkinsfile-extension-vsce

```diff
@@ -162,6 +159,17 @@ pipeline {
             }
         }
 
+        stage("Install vsce") {
+            when {
+                expression { "${run_job}" == "true" }
+            }
+            steps {
+                dir('vscode-inmanta') {
+                    sh 'npm install vsce'
```

Comment:
Since `vsce` is a dependency of the `vscode-inmanta` extension, I think it would be better to install the extension itself. When you use the `npm ci` command, it will 
1. clean the current node_modules 
2. install everything according to the `package-lock.json`, without changing anything, so the cleanup step shouldn't be necessary.

Reply:
That's interesting, I'll do that, thanks.